main/acle.md

```diff
@@ -1627,73 +1639,73 @@ be found in [[BA]](#BA).
-| `__ARM_PCS_AAPCS64` [[ssec-Pcs]](#procedure-call-standard)                                                                                                           | Arm PCS for AArch64.                                                                               | 1           |
-| `__ARM_PCS_VFP` [[ssec-Pcs]](#procedure-call-standard)                                                                                                               | Arm PCS hardware FP variant in use (32-bit-only)                                                   | 1           |
-| `__ARM_FEATURE_RNG` [[ssec-rng]](#random-number-generation-extension)                                                                                                | Random Number Generation Extension (Armv8.5-A)                                                     | 1           |
-| `__ARM_ROPI` [[ssec-Pic]](#position-independent-code)                                                                                                                | Read-only PIC in use                                                                               | 1           |
-| `__ARM_RWPI` [[ssec-Pic]](#position-independent-code)                                                                                                                | Read-write PIC in use                                                                              | 1           |
-| `__ARM_SIZEOF_MINIMAL_ENUM` [[ssec-Imptype]](#implementation-defined-type-properties)                                                                                | Size of minimal enumeration type: 1 or 4                                                           | 1           |
-| `__ARM_SIZEOF_WCHAR_T` [[ssec-Imptype]](#implementation-defined-type-properties)                                                                                     | Size of `wchar_t`: 2 or 4                                                                          | 2           |
-| `__ARM_WMMX` [[ssec-WMMX]](#wireless-mmx)                                                                                                                            | Wireless MMX extension (32-bit-only)                                                               | 1           |
-| `__ARM_FEATURE_MOPS` [[ssec-MOPS]](#memcpy-family-of-memory-operations-standarization-instructions---mops)                                                           | memcpy, memset, and memmove family of operations standardization instructions                      | 1           |
+| **Macro name**                                                                                                                                          | **Meaning**                                                                                        | **Example** |
```

Comment:
The changes in this table are hard to review (probably) because of the sorting. May I ask you to revert the sorting and submit it in a separate PR?

Reply:
Never mind - I looked at the individual commit and LGTM, no need to migrate it to a separate PR.